receiver/jaegerreceiver/testdata/config.yaml

```diff
@@ -1,7 +1,6 @@
 receivers:
   # The following demonstrates initializing the default jaeger receiver.
-  # By default all three protocols (grpc, thrift-http, thrift-tchannel)
-  # are enabled and the default endpoints are specified in factory.go
+  # By default this configuration enables nothing and will serve no jaeger protocols
```

Comment:
```suggestion
  # By default this configuration enables nothing and will serve no jaeger protocols.
```

Reply:
Please follow the convention in this file and end comment sentences with periods.

Reply:
Done.